compotime/models.py

```diff
@@ -256,6 +257,8 @@ def fit(self, y: pd.DataFrame, random_state: int = 0) -> Self:
         self.colnames_ = y.columns
         self.time_idx_ = y.index
 
+        y = _treat_zeros(y, threshold)
```

Comment:
I'm not sure if I would include this directly inside the model, instead of just keeping it as a preprocess function that can be applied when needed

Reply:
Let's discuss this, then we can merge